AMP-88309 Add additional property docs to Snowflake/BigQuery/S3/GCS (#94)
amplitude/amplitude-dev-center: docs/data/sources/snowflake.md

## Data fields

> You must include the mandatory fields for the data type when creating the SQL query. These tables outline the mandatory and optional fields for each data type. Other supported fields can be found in [HTTP V2 API documentation](/analytics/apis/http-v2-api/#keys-for-the-event-argument) (events) and [Identify API documentation](/analytics/apis/identify-api/#identification-parameter-keys) (user properties). Any column not in the list should be added to either `event_properties` or `user_properties`, otherwise it will get ignored.

Reviewer: ```suggestion
Include the mandatory fields for the data type when you create the SQL query. These tables outline the mandatory and optional fields for each data type. Find a list of other supported fields for events in the [HTTP V2 API documentation](/analytics/apis/http-v2-api/#keys-for-the-event-argument) and  for user properties in the [Identify API documentation](/analytics/apis/identify-api/#identification-parameter-keys). Add any columns not in those lists to either `event_properties` or `user_properties`, otherwise it's ignored. 
```

| `user_properties`               | No                                  | VARIANT (JSON Object)                | {"city":"chicago", "gender":"female"}         |
| `update_time_column`            | No (Yes if using time based import) | TIMESTAMP_NTZ                        | 2013-04-05 01:02:03.000                       |

> Other supported fields can be found in [HTTP V2 API documentation](/analytics/apis/http-v2-api/#upload-request-headers).

Reviewer: ```suggestion
Find other supported fields can in the [HTTP V2 API documentation](/analytics/apis/http-v2-api/#upload-request-headers).
```

### User properties

| Column name (must be lowercase) | Mandatory                           | Column data type      | Example                               |
| ------------------------------- | ----------------------------------- | --------------------- | ------------------------------------- |
| `user_id`                       | Yes                                 | VARCHAR               | datamonster@gmail.com                 |
| `user_properties`               | Yes                                 | VARIANT (JSON Object) | {"city":"chicago", "gender":"female"} |
| `update_time_column`            | No (Yes if using time based import) | TIMESTAMP_NTZ         | 2013-04-05 01:02:03.000               |

> Other supported fields can be found in [Identify API documentation](/analytics/apis/identify-api/#identification-parameter-keys).

Reviewer: ```suggestion
Find other supported fields in the [Identify API documentation](/analytics/apis/identify-api/#identification-parameter-keys).
```
